demoapp/www/js/crashes.js

```diff
@@ -0,0 +1,153 @@
+﻿$(document).bind('pageinit', function () {
+
```

Comment:
nit: don't think if we really need this blank lines at the beginning of each file, I would prefer to remove them.